packages/ilios-common/addon/components/course/rollover.hbs

```diff
@@ -73,16 +70,18 @@
         <div class="item">
           <label>
             {{t "general.startDate"}}:
-            <br>
+            <br />
             <small>
               ({{t "general.mustBeSameDayOfWeekAsCurrentStartDate"}})
             </small>
           </label>
-          <Course::RolloverDatePicker
-            @course={{@course}}
-            @value={{this.startDate}}
-            @onChange={{set this "startDate"}}
-          />
+          {{#if this.allCourses}}
```

Comment:
If `this.allCourses` what? It's working off of the null and that's valid, but it's hard to read it here. Would prefer `if this.allCoursesData.isResolved`.

Reply:
Changing it to `{{#if this.allCoursesData.isResolved}}` just makes the DatePicker not show up at all. If I change it to `{{#if (is-array this.allCourses)}}`, like the usage for the other `<select>` that holds `this.years`, then it works. It seems they both check to make sure it's not null.

Reply:
After changing the getters, this is now serviced by the `{{#if this.allCoursesData.isResolved}}`.